Moving cep-47 tutorial to the repository

Buddhika-Pathirathna opened the pull request:
**Related links**
Card - [281](https://github.com/casper-network/docs/issues/281)

**Changes**
Added CEP-47 tutorial content from the developer portal to the README.md file in [cep-47 repository](https://github.com/casper-ecosystem/casper-nft-cep47)

**Notes**
Added a separate file named TUTORIAL.md and linked that file to README.md
Added a new folder to include images
Did not remove the tutorial folder from the docs repository yet
Adjusted certain links to route to the Casper Network docs site. E.g. [Getting Started Guide] (https://casper.network/docs/dapp-dev-guide/getting-started/)

ipopescu reviewed (changes requested):
@Buddhika-Pathirathna Thank you.

I think we should move "Contract Interaction and Events" to another file. This file is getting too long. Also, structurally, it can be a different tutorial. You can call it Contract-Interaction-Tutorial.md. You can rename TUTORIAL.md to Basic-Tutorial.md.

I know this content is already live, but I saw some improvements.
- I updated the flow because there was repetition when putting everything in one file. Please review the new flow and double-check all the links.
- I moved the information about the standard in the README and fixed a broken link 
- I rewrote all references to https://github.com/casper-ecosystem/casper-nft-cep47
since you are already in the repo and some of the wording did not make sense
- Replaced deployment with “sending the contract to the network”, we try to avoid this verb “deployment” now
- Also note: CEP-47 is a standard, so it is best to say “Casper NFT contract (that follows the CEP-47 standard)”, rather than “CEP-47 contract”. Let’s not use this in the future “ a.k.a. CEP-47 smart contract”. I rewrote such references.
- Updated formatting; example: 
  - Added a break after </details> like this `<br></br>`
   - In some sections, we used numbers such as **2. Sending the deploy to the network** and in others not, made this consistent
   - Checked all the headings. We should use the top heading with one `#` only once. For `####` headings, let’s use bold to stand out
- Step 6 is removed because it is the same as step 5 (installing the contract). This was a mistake in the original content.

Please review. Thanks!

Buddhika-Pathirathna commented:
> @Buddhika-Pathirathna Thank you.
> 
> I think we should move "Contract Interaction and Events" to another file. This file is getting too long. Also, structurally, it can be a different tutorial. You can call it Contract-Interaction-Tutorial.md. You can rename TUTORIAL.md to Basic-Tutorial.md.
> 
> I know this content is already live, but I saw some improvements.
> 
> * I updated the flow because there was repetition when putting everything in one file. Please review the new flow and double-check all the links.
> * I moved the information about the standard in the README and fixed a broken link
> * I rewrote all references to https://github.com/casper-ecosystem/casper-nft-cep47
>   since you are already in the repo and some of the wording did not make sense
> * Replaced deployment with “sending the contract to the network”, we try to avoid this verb “deployment” now
> * Also note: CEP-47 is a standard, so it is best to say “Casper NFT contract (that follows the CEP-47 standard)”, rather than “CEP-47 contract”. Let’s not use this in the future “ a.k.a. CEP-47 smart contract”. I rewrote such references.
> * Updated formatting; example:
>   
>   * Added a break after  like this `<br></br>`
>   * In some sections, we used numbers such as **2. Sending the deploy to the network** and in others not, made this consistent
>   * Checked all the headings. We should use the top heading with one `#` only once. For `####` headings, let’s use bold to stand out
> * Step 6 is removed because it is the same as step 5 (installing the contract). This was a mistake in the original content.
> 
> Please review. Thanks!

Thanks a lot, @ipopescu for performing a solid review on this. I will address those points and update the branch.
